src/tcgwars/logic/impl/gen7/SunMoonPromos.groovy

```diff
@@ -2908,12 +3201,122 @@ public enum SunMoonPromos implements LogicCardInfo {
           }
         }
         };
+      case LEAFEON_SM237:
+        return evolution (this, from:"Eevee", hp:HP110, type:G, retreatCost:2) {
+          weakness R
+          move "Aromax", {
+            text "Heal all damage from 1 of your Benched Pokémon."
+            energyCost G
+            attackRequirement {
+              assert my.bench : "You have no Benched Pokémon."
+              assert my.bench.find({ it.numberOfDamageCounters }) : "None of your Benched Pokémon are damaged."
+            }
+            onAttack {
+              def benchToHeal = my.bench.findAll { it.numberOfDamageCounters }.select("Select a Benched Pokémon to heal all damage from.")
+                if (benchToHeal) {
+                healAll benchToHeal
+}
+            }
+          }
+          move "Leaf Blade", {
+            text "80+ damage. Flip a coin. If heads, this attack does 40 more damage."
+            energyCost G, C, C
+            onAttack {
+              damage 80
+              flip { damage 40 }
+            }
+          }
+        };
+      case GLACEON_SM238:
+        return evolution (this, from:"Eevee", hp:HP110, type:W, retreatCost:2) {
+          weakness M
+          move "Snow Cloak", {
+            text "30 damage. Flip a coin. If heads, prevent all effects of attacks, including damage, done to this Pokémon during your opponent's next turn."
+            energyCost W
+            attackRequirement {}
+            onAttack {
+              damage 30
+              flip { peventAllEffectsNextTurn }
```

Comment:
Looks like this method is misspelt and also it is missing the `( )` at the end

Reply:
Fixed the issue